pandas/tests/io/conftest.py

```diff
@@ -2,30 +2,34 @@
 
 import pytest
 from pandas.io.parsers import read_table
+from pandas.util import testing as tm
 
-HERE = os.path.dirname(__file__)
 
+@pytest.fixture
+def parser_data(request):
+    return os.path.join(tm.get_data_path(), '..', 'parser', 'data')
 
-@pytest.fixture(scope='module')
-def tips_file():
+
+@pytest.fixture
+def tips_file(parser_data):
     """Path to the tips dataset"""
-    return os.path.join(HERE, 'parser', 'data', 'tips.csv')
+    return os.path.join(parser_data, 'tips.csv')
 
 
-@pytest.fixture(scope='module')
-def jsonl_file():
+@pytest.fixture
+def jsonl_file(parser_data):
     """Path a JSONL dataset"""
-    return os.path.join(HERE, 'parser', 'data', 'items.jsonl')
+    return os.path.join(parser_data, 'items.jsonl')
 
 
-@pytest.fixture(scope='module')
-def salaries_table():
+@pytest.fixture
+def salaries_table(parser_data):
     """DataFrame with the salaries dataset"""
-    path = os.path.join(HERE, 'parser', 'data', 'salaries.csv')
+    path = os.path.join(parser_data, 'salaries.csv')
     return read_table(path)
 
 
-@pytest.fixture(scope='module')
+@pytest.yield_fixture
```

Comment:
I think `yield_fixture`s are deprecated. You can just `yield` in a regular fixture.

Reply:
hahah didn't know that!